library/nodeping.py

```diff
@@ -810,6 +834,17 @@ def run_module():
                 msg="Failed to delete checkid %s" % params["checkid"], **output
             )
 
+    elif action == "disable":
+        if not params["target"]:
```

Comment:
Here we should use `checkid` as well to be consistent with the API

Reply:
It does not work with checkid.